build-conf/Cobol.properties

```diff
@@ -45,11 +45,11 @@ cobol_loadOptions=cyl space(1,1) dsorg(PO) recfm(U) blksize(32760) dsntype(libra
 cobol_tempOptions=cyl space(5,5) unit(vio) blksize(80) lrecl(80) recfm(f,b) new
 cobol_printTempOptions=cyl space(5,5) unit(vio) blksize(133) lrecl(133) recfm(f,b) new
 
-cobol_testsrcDatasets=${cobol_testcase_srcPDS}
-cobol_testsrcOptions=cyl space(1,1) lrecl(80) dsorg(PO) recfm(F,B) dsntype(library)
+cobol_test_srcDatasets=${cobol_testcase_srcPDS}
```

Comment:
we could need to update the readme.md for the build config as well https://github.com/IBM/dbb-zappbuild/tree/development/build-conf